### PR TITLE
get hamka data from website not printed menu

### PR DESCRIPTION
**Pull request by Zygro**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -27,10 +28,13 @@
...
-    "babel-cli": "^6.24.1",
+    "@babel/cli": "^7.1.2",
+    "@babel/core": "^7.1.2",
+    "@babel/plugin-transform-runtime": "^7.1.0",
+    "@babel/preset-env": "^7.1.0",
+    "@babel/runtime": "^7.1.2",
     "babel-eslint": "^7.2.3",
     "babel-plugin-transform-runtime": "^6.23.0",
-    "babel-preset-es2015": "^6.16.0",
     "babel-register": "^6.26.0",
```
Comment (misogl) [inline package.json]: We should switch all babel packages to scoped names (`@babel/register`).

**Review by misogl — CHANGES_REQUESTED** [review]
Please also put the new URL for hamka menu to `.env.template` (`OBEDBOT_HAMKA` env variable).

File: .eslintrc [hunk 2]
```diff
@@ -4,6 +4,14 @@
...
     "import/ignore": ["node_modules", "assets", "\\.(scss|less|css)$"],
   },
+  "parserOptions": {
+    "ecmaVersion": 6,
+    "sourceType": "module",
+    "ecmaFeatures": {
+      "modules": true,
+      "jsx": true
+    }
+  },
```
Comment (Zygro) [inline .eslintrc:14]: added this because of https://github.com/eslint/eslint/issues/4344

**Review by conqeror — APPROVED** [review]

**Comment by misogl** [comment]
The hamka menu is now fixed. If it stops working again we can fix it by merging this PR.

**Pull request closed without merging**